prac_05/wimbledon.py

```diff
@@ -0,0 +1,58 @@
+"""
+wimbledon
+Estimate: 60 minutes
+Actual: 70 minutes
+"""
+
+
+def main():
+    """Get and print the number of wins per champion and the countries of champions in alphabetical order"""
+    filename = "wimbledon.csv"
+
+    countries, names = get_champion_information(filename)
+    champion_to_wins = get_number_of_wins_per_champion(names)
+    countries_alphabetical = get_countries_in_alphabetical_order(countries)
+
+    # print results
+    max_length = get_max_length_for_printing(champion_to_wins)
+    print("Wimbledon Champions:")
+    for name, wins in champion_to_wins.items():
+        print(f"{name:{max_length}} - Wins: {wins:2}")
+    print(f"These 12 countries have won Wimbledon:\n {countries_alphabetical}")
```

Comment:
Perhaps add a variable name in place of hardcoded '12'

Reply:
Done.